problems/objects.js

```diff
@@ -226,16 +358,25 @@ const updateList = () => {};
  * @returns {Object} The Object without the key.
  */
 
- const deleteKey = () => {};
-
+ const deleteKey = (obj,string) => {
+   for(key in obj){
+   if(key === string){
+    delete obj[key]
```

Comment:
Do we need a for loop and an if statement if we are just trying to key into an object?